MAINTAINERS.md

```diff
@@ -89,31 +89,40 @@ A simple majority vote is required to authorize a SPIRE release at a specific co
 This section summarizes the steps necessary to execute a SPIRE release. Unless explicitly stated, the below steps must be executed in order.
 
 The following steps must be completed one week prior to release:
-* Ensure all changes intended to be included in the release are fully merged
-* Identify a specific commit as the release candidate
-* Raise an issue "Release SPIRE X.Y.Z", and include both the release candidate commit hash and the proposed changelog updates
+* Ensure all changes intended to be included in the release are fully merged.
+* Identify a specific commit as the release candidate.
+* Create a draft pull request against master branch with the updates to the CHANGELOG following [these guidelines](doc/changelog_guidelines.md). This allows those tracking the project to have early visibility into what will be included in the upcoming release and an opportunity to provide feedback. The release date can be set as "TBD" while it is a draft.
+* Raise an issue "Release SPIRE X.Y.Z", and include the release candidate commit hash. Reference the pull request with the updates to the CHANGELOG.
+* If the current state of the master branch has diverged from the candidate commit due to other changes than the ones from the CHANGELOG:
+  * If there is not a version branch for this release, create a branch following the guidelines described in [Version branches](#version-branches).
+  * Create a GitHub project named `Release vX.X.X` to identify the PRs that will be cherry-picked. The project should have two statuses to track the progress: one to identify the PRs to be cherry-picked and one for those that have been merged in the version branch.
+  * Make sure that the [version in the branch](pkg/common/version/version.go) has been bumped to the version that is being released.
+  * Cherry-pick into the version branch the commits for all the changes that must be included in the release, including the updates in the CHANGELOG and update the corresponding status in the project.
```

Comment:
At this point in time, the CHANGELOG updates have not been merged. It also has the `TBD` date still.

Reply:
This is a good point. The updates in the CHANGELOG cannot be cherry-picked since they have not been merged at this point. This should be part of the steps that are done later, as part of marking the PR "Ready for review" and merging it. I think it's worth trying to fix it now. I'll try.